test/functional/apps/discover/group3/_request_counts.ts

```diff
@@ -223,7 +223,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
     });
 
-    describe('ES|QL mode', () => {
+    // @TODO: fix this in a follow up
```

Comment:
I didn't realize this was going into a feature branch at first and was surprised to see these tests skipped, but assuming they'll be addressed and unskipped before making it to `main`, this LGTM 👍

Reply:
Davis correct, we found some performance problems so this is not going to be merged. Also autocomplete doesnt work as expected. But as this is a big PR already we want to move this into a feature branch and then continue with the investigation and the development